sensing/pointcloud_preprocessor/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ if(NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_EXTENSIONS OFF)
 endif()
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  add_compile_options(-Wall -Wextra -Wpedantic -Werror)
+  add_compile_options(-Wall -Wextra -Werror)
```

Comment:
Why did you remove `-Wpedantic`?

Reply:
`/usr/include/pcl-1.10/pcl/octree/octree_key.h:155:9: error: ISO C++ prohibits anonymous structs [-Werror=pedantic]`
This causes the error above. I don't have any idea why `-Wno-gnu-anonymous-struct` doesn't work...

Reply:
I see. Hmm... :thinking: